addon/ng2/blueprints/ng2/files/package.json

```diff
@@ -2,6 +2,9 @@
   "name": "<%= htmlComponentName %>",
   "version": "0.0.0",
   "license": "Apache-2.0",
+  "scripts":{
+    "postinstall": "webdriver-manager update"
```

Comment:
I actually do have a question about that. I'm partial to all-local dependencies (bower/karma/protractor) because that way each project can be version locked, and would generally only need node installed to run (and no global packages at all). Doing things this way will undoubtedly add time to the initial startup though.

So right now, a project generated in this way will have it's own protractor and own webdriver, thus the `webdriver-manager update` is necessary to be ran for each project.

What's your view on this?